server/persistence/mysql/games.go

```diff
@@ -0,0 +1,569 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+
+	"github.com/joshprzybyszewski/cribbage/jsonutils"
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/joshprzybyszewski/cribbage/server/persistence"
+)
+
+const (
+	// Games stores the state of a game at a given time.
+	//   Each Action will update a games state and we keep a full history of all actions.
+	// The columns act as follows:
+	// GameID is a UUID to identify a game
+	// NumActions is how many actions have occurred in the game before this one
+	// ScoreBlue, ScoreRed, and ScoreGreen are the scores for each color
+	// ScoreBlueLag, ScoreRedLag, and ScoreGreenLag are the previous scores for each color
+	// Phase is the model.Phase that the game is currently in
+	// CutCard is a number representation of the card that's been cut
+	// nolint:lll Crib is an 8-byte int of the (up to 4) cards in the crib where every two bytes is each crib card. Tricky, I know. Probably shouldn't do it this way, but yolo, this is a fun project and I can save space and mental pain
```

Comment:
Do we actually need two bytes to represent a card? Could we use the lower 6 bits as the value of the card and the upper 2 as the suit? Maybe this is too much trouble to save 4 bytes though :shrug:

Reply:
Update: I did make the space savings to only use 4 bytes (`INT` is 4 bytes, `BIGINT` is 8 bytes), so this is slightly out of date. will update.